Source/JavaScriptCore/assembler/X86Assembler.h

```diff
@@ -4390,6 +4384,27 @@ class X86Assembler {
                 thirdByte |= (~inOpReg & 0xf) << 3;
                 putByteUnchecked(thirdByte);
             }
+
+            ALWAYS_INLINE void threeBytesVex(bool isVEX256, OneByteOpcodeID simdPrefix, VexImpliedBytes impliedBytes, bool isW1, RegisterID r, RegisterID x, RegisterID b)
```

Comment:
assert !isVEX256?

Reply:
This function is designed as a universal help method for VEX prefix encoding.